control-base/src/main/java/fi/nls/oskari/control/feature/AbstractFeatureHandler.java

```diff
@@ -132,7 +132,8 @@ protected Feature getFeature(JSONObject jsonObject) throws ActionParamsException
             }
             JSONArray data = geometries.getJSONArray("data");
             Geometry geometry = getGeometry(geometryType, data, getSrid(srsName, 3067));
-            if(ProjectionHelper.isFirstAxisNorth(crs)) {
+            //TODO: ProjectinHelper doesn't function like it should
+            if(ProjectionHelper.isFirstAxisNorth(crs) || srsName.equals("EPSG:3879")) {
```

Comment:
Please change this to something like PropertyUtil.getOptional("actionhandler.AbstractFeatureHandler.forceXY", false) and add `actionhandler.AbstractFeatureHandler.forceXY=true` to oskari-ext.properties where needed.